pydsa/counting_inversions.py

```diff
@@ -0,0 +1,61 @@
+# Counting Inversions Using Merge Sort
+# Complexity: O(nlog(n))
+
+def get_divide(a):
+    s = len(a)
+    L=[]
```

Comment:
space here as per PEP8

Also it would be better to name variables that make sense to people reading your code.

`Code is read much more times than it is written.`